src/screens/Assistant/Assistant.tsx

```diff
@@ -537,6 +548,13 @@ const useStyles = StyleSheet.createThemeHook((theme) => ({
   fadeChild: {
     marginVertical: theme.spacings.medium,
   },
+  loadingIndicator: {
+    marginTop: theme.spacings.xLarge,
+    flexDirection: 'row',
+  },
+  loadingText: {
+    marginTop: theme.spacings.xLarge * 2,
```

Comment:
Basert på videoen synes jeg `xLarge` er nok her. Generelt skal det være velbegrunnet hvis man beveger seg utover default-spacingene som er i designsystemet. Men det er ikke veldig vanskelig å overbevise meg om at 48 er riktig her hvis du mener det i dette tilfeller er verdt det 😅

Reply:
Det er spinneren på android som tar mye plass, slik at den havner over teksten ellers. Enig i at det ser for mye ut på iOS, men blir feil på android ellers. 
![Screenshot 2022-03-25 at 12 50 07](https://user-images.githubusercontent.com/1774972/160115701-df39f046-d7cf-48cb-a7ea-18016b881319.png)